policy/README.md

```diff
@@ -25,9 +27,9 @@ One of the major output of the team effort is the [cloud native policy
 white paper](https://docs.google.com/document/d/1StDYW1zHVSF1Qswk0ScsyKw766AbAHOikyCNtqCsMMY/edit?usp=sharing)
 
 The current top priority development work are 
-- [policy formal verification](./overview-policy-formal-verification.md)
+- [Policy Formal Verification](/policy/overview-policy-formal-verification.md)
 - [Policy Violation CRD](https://docs.google.com/document/d/1QJWcaJdo8w88tIixiZp9zBMHKghTMrHTyHyk6xcoVhk/edit?usp=sharing)
 
 ## Contact
-The team could be reached via either k8s slack channel #wg-policy or cncf slack
-channel #sig-security.
+The team can be reached via either kubernetess slack channel #wg-policy or
```

Comment:
caps on Kubernetes and CNCF should be consistent in the doc as well

Reply:
done